### ISSUE TITLE
Merge Analyze & Check Style

### ISSUE THREAD

**Issue by zrzka** [omz/Pythonista-Issues#357]
This issue renders #350, #349, #352 as obsolete and #351 (``editor.analyze``) can do both things.

I just released _Analyze & Check Style_ in Black Mamba. What it does? One shortcut for:

* `pyflakes` analysis,
* `pep8` code style check.

We can end up with more than one annotation on a line. That's okay, I'm displaying it in this way:

* all line annotations are sorted by priority (pyflakes > pep8, error > warning)
* first sorted annotation style is used
* annotation text is simple `',\n'.join()` of all annotations

Also it should:

* scroll to the first issue,
* do not show HUD if there're issues, it's useless because of scrolling

Check attached screenshot:

![img_0091](https://user-images.githubusercontent.com/1084172/29656169-dca3799a-88b3-11e7-9fcb-c66ce106764a.jpg)

**Comment by cclauss** [comment]
http://flake8.pycqa.org is a wrapper around both pyflakes and pep8 for other platforms.

https://github.com/openstack-dev/hacking is a wrapper around flake8 adding even more error checks.

**Comment by zrzka** [comment]
I checked Pythonista frameworks and I didn't find flake8. So I decided to go with pyflakes & pep8, which are available (not forcing users to install dependencies). But I agree it would be nice if we can have #85 fixed and flake8 will be used for `editor.analyze` / Wrench & Analyze.

**Comment by cclauss** [comment]
Correct.  Flake8 is not currently part of Pythonista so your choice of pyflakes & pep8 was the right way to go.
